Python/Python.sublime-syntax

```diff
@@ -328,29 +328,67 @@ contexts:
       scope: constant.language.python
 
   numbers:
-    - match: \b(?i)(0x\h*)L # py2
+    # https://docs.python.org/3/reference/lexical_analysis.html#numeric-literals
+    # hexadecimal
+    - match: \b(?i)(?:0x\h*)(L) # py2
```

Comment:
There's some redundancy in having matches check for the leading `0x` constantly, but it's just a suggestion and if you feel that it isn't worth pursuing this idea then don't feel obliged.